components/Members.tsx

```diff
@@ -0,0 +1,43 @@
+import { allAuthors } from 'contentlayer/generated'
+import Link from '@/components/Link'
+import Image from '@/components/Image'
+import { useState, useEffect } from 'react'
+
+export default function Members({ subset }) {
+  const [sortedMembers, setSortedMembers] = useState([...allAuthors])
+
+  useEffect(() => {
+    setSortedMembers(allAuthors.sort(() => 0.5 - Math.random()))
+  })
+
+  let members
+  switch (subset) {
```

Comment:
Nice! I like that. 

Note that there's also the `design` team, see `DesignTeam.tsx` which is included as an MDX component in https://opensats.org/nostr#nostr-design - but I'd say that's a story for a follow-up PR.